docker/Dockerfile.kubectl

```diff
@@ -1,11 +1,25 @@
+# BASE_DISTRIBUTION is used to switch between the old base distribution and distroless base images
+ARG BASE_DISTRIBUTION=default
+
 # hadolint ignore=DL3006
-FROM istionightly/base_debug
-# Image for post install jobs
+FROM istionightly/base_debug as default
 
 # This container should only contain kubectl.  Hard-coding to use Linux K8s 1.11.1 version.
 ADD https://storage.googleapis.com/kubernetes-release/release/v1.11.1/bin/linux/amd64/kubectl /usr/bin/kubectl
```

Comment:
as a followup we may want to bump the version, we only support k8s 1.12-1.15 now